### ISSUE TITLE
HTML for articles contains wrong links

### ISSUE THREAD

**Issue by fsundermeyer** [openSUSE/suse-xsl#240]
Generate an HTML file for the DAPS quickstart. The "index" file is named art.daps.quick.html, however, a few lilnks point to "book.daps.quick.html":

    <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Quick Start</title><meta name="generator" content="DocBook XSL Stylesheets V1.78.1" /><meta name="keywords" content="DAPS, DocBook Authoring and Publishing Suite, DocBook, XML, Tool chain, Quick Start" /><meta name="keywords" content="DAPS, DocBook Authoring and Publishing Suite, DocBook, XML, Tool chain, Command Line" /><link rel="home" href="index.html" title="DAPS Documentation" /><link rel="up" href="book.daps.quick.html" title="Quick Start" /><link rel="prev" href="book.daps.quick.html" title="Quick Start" /><link rel="next" href="sec.daps.quick.dbversions.html" title="Supported DocBook Versions" /></head><body><div class="navheader"><table width="100%" summary="Navigation header"><tr><th colspan="3" align="center">Quick Start</th></tr><tr><td width="20%" align="left"><a accesskey="p" href="book.daps.quick.html">Prev</a>

    </p></div></div><div class="navfooter"><table width="100%" summary="Navigation footer"><tr><td width="40%" align="left"><a accesskey="p" href="book.daps.quick.html">Prev</a> </td><td width="20%" align="center"><a accesskey="u" href="book.daps.quick.html">Up</a></td><td width="40%" align="right"> <a accesskey="n" href="sec.daps.quick.dbversions.html">Next</a></td></tr><tr><td width="40%" align="left" valign="top">Quick Start </td><td width="20%" align="center"><a accesskey="h" href="index.html">Home</a></td><td width="40%" align="right" valign="top"> Supported DocBook Versions</td></tr></table></div></body></html>

**Comment by sknorr** [comment]
Indeed, Frank is right: The OLDER stylesheets do that. Apparently, there is a hard-coded "book" hiding somewhere.

**Comment by janajaeger** [comment]
This is still not fixed. Confirming Stefan's suspicions. Let's hunt for the hardcoded "book.". @tomschr let's have a short chat about this some time.
